### PR TITLE
Bug 2010662: Wait until cluster operators recover before proceeding

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

**Comment by wewang58** [comment]
Now in wip status

**Comment by wewang58** [comment]
@dmage saw e2e-aws-operator job failed, but operator part success, seems prow issue,right? 
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-image-registry-operator/726/pull-ci-openshift-cluster-image-registry-operator-master-e2e-aws-operator/1458993907726553088

File: test/framework/cvo.go [hunk 1]
```diff
@@ -70,6 +70,15 @@ func DisableCVOForOperator(te TestEnv) {
 	})
 	changed = changed || componentChanged
 
+	cv.Spec.Overrides, componentChanged = addCompomentOverride(cv.Spec.Overrides, configv1.ComponentOverride{
```
Comment (flavianmissi) [inline test/framework/cvo.go:73]: Would be nice with a comment here providing some background on why we're disabling MCO.
Wdyt?

**Comment by dmage** [comment]
@wewang58 no, that's not a prow issue. Almost every test has failed, either there is something wrong with the cluster or with tests. Let's rut it again.

/test e2e-aws-operator

**Comment by dmage** [comment]
Screen Shot 2021-11-12 at 18 33 43

**Comment by dmage** [comment]
CVO has fixed a bug https://github.com/openshift/cluster-version-operator/issues/688, so I also updated our tests to account it.

**Comment by flavianmissi** [comment]
Thanks for fixing 🚀 

/lgtm

**Pull request merged**